README.md

```diff
@@ -1,2 +1,18 @@
-# Hopp JS
-The official website for Hopp JS
\ No newline at end of file
+# hoppjs.com
+
+This is the repository for the official website for Hopp JS available at [hoppjs.com](http://hoppjs.com).
```

Comment:
Keep `hopp` in lowercase at all times, and we'll just call it hopp instead of hoppjs. It's just that `hopp` is taken on Github & as a `.com` domain.